labs.yml

```diff
@@ -46,3 +46,10 @@ commands:
       - name: report
         default: all
         description: Report type
+  - name: generate-recon-config
```

Comment:
just keep validate-recon-config related in this PR

Reply:
Done, removed all other code